EIPS/eip-2028.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 2028
+title: Calldata gas cost reduction
+author: Alexey Akhunov(@AlexeyAkhunov), Eli Ben Sasson (eli@starkware.co), Tom Brand (tom@starkware.co), Avihu Levy (avihu@starkware.co)
+discussions-to: https://ethereum-magicians.org/t/eip-2028-calldata-gas-cost-reduction/3280
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-05-03
+---
+
+## Simple Summary
+We propose to reduce the gas cost of Calldata (`GTXDATANONZERO`) from its current value of 68 gas per byte to a lower cost, to be backed by mathematical modeling and empirical estimates. The mathematical model is the one used in the works of Sompolinsky and Zohar [1] and Pass, Seeman and Shelat [2], which relates network security to network delay. We shall (1) evaluate the theoretical impact of lower Calldata gas cost on network delay using this model, (2) validate the model empirically, and (3) base the proposed gas cost on our findings.
+
+## Motivation
+There are a couple of main benefits to accepting this proposal and lowering gas cost of Calldata
+On-Chain Scalability: Generally speaking, higher bandwidth of Calldata improves scalability, as more data can fit within a single block.
```

Comment:
I think marking these bullet points may increase readability when the markdown is formatted.

Reply:
Agree, somehow missed that. Thank you.